core/cli/src/informant/display.rs

```diff
@@ -0,0 +1,149 @@
+// Copyright 2017-2019 Parity Technologies (UK) Ltd.
```

Comment:
2019

Reply:
Is that true even if it's just extracted code?